### PR TITLE
Document Wrangler Generate Command

### PR DESCRIPTION
**Pull request by lauragift21**
`wrangler generate` was added in v2.0.24, this PR documents the command in Wrangler docs.

https://03425558.cloudflare-docs-7ou.pages.dev/workers/wrangler/commands/

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -61,6 +62,25 @@ $ wrangler init [NAME] [-y / --yes]
 
 ---
 
+## generate
+
+Create a Wrangler project using existing Worker's template.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
Create a Wrangler project using an existing [Workers template](https://github.com/cloudflare/worker-template).
```

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ weight: 2
 Wrangler offers a number of commands to manage your Cloudflare Workers.
 
 - [`init`](#init) - Create a skeleton Wrangler project, including the `wrangler.toml` file.
+- [`generate`](#generate) - Create a Wrangler project using existing Worker's template.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
- [`generate`](#generate) - Create a Wrangler project using an existing [Workers template](https://github.com/cloudflare/worker-template).
```

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -61,6 +62,25 @@ $ wrangler init [NAME] [-y / --yes]
...
+```sh
+$ wrangler generate [name] [template]
+```
+
+{{<definitions>}}
+
+- `name` {{<type>}}string{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}} {{<prop-meta>}}(default: name of working directory){{</prop-meta>}}
+  - The name of the Workers project. This is both the directory name and `name` property in the generated `wrangler.toml` [configuration](/workers/wrangler/configuration/) file.
+- `template` {{<type>}}string{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}} 
+  - The URL of a GitHub template, with a default [worker-template](https://github.com/cloudflare/worker-template"). Browse a list of available templates on [cloudflare/templates](https://github.com/cloudflare/templates) repository.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
  - The URL of a GitHub template, with a default [worker-template](https://github.com/cloudflare/worker-template). Browse a list of available templates on [cloudflare/templates](https://github.com/cloudflare/templates) repository.
```

**Review by deadlypants1973 — COMMENTED** [review]
PCX review

**Review by deadlypants1973 — APPROVED** [review]

**Comment by kodster28** [comment]
Just fixed our failing builds with #5539, so if you want to fix the build errors you'll need to merge in production

**Comment by lauragift21** [comment]
> Just fixed our failing builds with #5539, so if you want to fix the build errors you'll need to merge in production

Oh I was wondering if something was wrong. Alright will do that

**Pull request merged**